bin/install

```diff
@@ -23,6 +23,7 @@ TMP_DOWNLOAD_DIR="$(mktemp -d -t "asdf_${toolname}_XXXXXX")"
 readonly TMP_DOWNLOAD_DIR
 trap 'rm -rf "${TMP_DOWNLOAD_DIR?}"' EXIT
 
+SKIP_VERIFY=${ASDF_HASHICORP_SKIP_VERIFY:-"false"}
```

Comment:
Can `ASDF_HASHICORP_SKIP_VERIFY` be documented somewhere?  The top-level README, maybe?